src/components/layout/nav.js

```diff
@@ -295,6 +295,7 @@ const SignupButton = styled(Button)`
 `
 
 const LinkSignupButton = styled(LinkButton)`
+    white-space: nowrap;
```

Comment:
Doing this will break some buttons when we have unexpected long translations.

Reply:
Sure. Without this property button more adaptive. Thought that we need everything on one line as in design. After checking other languages noticed that a lot of buttons are in 2 lines